test/filters/browser_tests.filter

```diff
@@ -753,7 +753,6 @@
 -CaptivePortalBlockingPageTest.*
 -CaptivePortalBrowserTest.*
 -CertificateReportingServiceBrowserTest.*
--CertificateTransparencyPolicyTest.CertificateTransparencyEnforcementDisabledForUrls
```

Comment:
@mkarolin You disabled this test in https://github.com/brave/brave-core/commit/f7514996c2373a1d0752d96b4059c5a94ce982db. Do you remember whether it was because it failed or because it was flaky?

It seems to work for me now that I've enabled the underlying feature, but maybe we should still leave it disabled because it's flaky?